src/ompl/base/samplers/informed/src/PathLengthDirectInfSampler.cpp

```diff
@@ -115,21 +115,24 @@ namespace ompl
             }
             else if (InformedSampler::space_->isCompound())
             {
+              std::cout << "PathLengthDirectInfSampler: Compound StateSpace. State Space Type: " << InformedSampler::space_->getType() << std::endl;
```

Comment:
there must be logging support in ompl i'm assuming?

Reply:
i don't think there's any that works, a lot of the ompl log lines i see are commented out... i'll remove all the std::out lines when we're ready to merge because they're really verbose and only useful for actively debugging ompl anyways